engine_test_impl.go

```diff
@@ -0,0 +1,23 @@
+package wapc
+
+// __attribute__((weak))
+// extern unsigned long long meteringFn(unsigned int op);
```

Comment:
ideally running the main tree tests can be done without CGO also. While we aren't doing it right now, one thing this allows is proving other archs work by building a test binary and running on scratch containers (wazero does this for riscv for example)

Reply:
file removed